src/Apps/Artist/Routes/AuctionResults/Components/AuctionResultHeader.tsx

```diff
@@ -1,21 +1,38 @@
-import { Box, Sans, Serif } from "@artsy/palette"
+import { Box, Link, Sans, Serif } from "@artsy/palette"
 import { AuctionResultHeader_artist } from "__generated__/AuctionResultHeader_artist.graphql"
 import React from "react"
 import { createFragmentContainer } from "react-relay"
 import { graphql } from "relay-runtime"
+import { data as sd } from "sharify"
 
 interface Props {
   artist: AuctionResultHeader_artist
 }
 
 const AuctionResultHeader: React.FC<Props> = props => {
   const { artist } = props
+
   return (
     <Box pb={2}>
       <Sans size="5t">Auction results</Sans>
       <Serif size="3" color="black100">
-        Filter past auction results to compare and evaluate {artist.name}'s
-        market.
+        Filter auction results to compare past lots by medium, size, and more.
+        Note that auction prices vary based on market specifics at the time of
+        the auction and may not be indicative of the current gallery market. To
+        get the best sense of value, pair the artist’s auction results with
+        their{" "}
+        <Link href={`${sd.APP_URL}/artist/${artist.slug}/cv`}>
```

Comment:
Instead of the `Link` component, we should use the `RouterLink` component [as seen here](https://github.com/artsy/reaction/blob/ff7a580c8d3b63f19a81d36664a615b1f64a5683/src/Apps/Artist/Components/ArtistHeader.tsx#L155). Using `RouterLink` will cause a client side navigation event to happen rather than reloading a whole new page. For newer pages (artist, artwork, order, collections, etc) that's our preferred method of routing.

Reply:
Interesting to learn the different handlings of each component